### PR TITLE
Added blockboy/ac980mini keyboard folder

### PR DESCRIPTION
**Pull request by rooski15**
## Description

Built keyboard folder to support the Block Boy "AC980Mini" keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/blockboy/ac980mini/readme.md
Comment (drashna) [inline keyboards/blockboy/ac980mini/readme.md:4]:
```suggestion
![AC980mini kle](https://imgur.com/prgMVo2h.jpg)

```

File: keyboards/blockboy/ac980mini/readme.md
Comment (drashna) [inline keyboards/blockboy/ac980mini/readme.md:3]: Please rehost this image on an image hosting site such as imgur.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/blockboy/ac980mini/readme.md
Comment (rooski15) [reply keyboards/blockboy/ac980mini/readme.md]: Done!

File: keyboards/blockboy/ac980mini/ac980mini.c
Comment (drashna) [inline keyboards/blockboy/ac980mini/ac980mini.c:60]: This is deprecated and should be removed, regardless if it's actively being used or not. 
```suggestion
```

File: keyboards/blockboy/ac980mini/rules.mk
Comment (drashna) [inline keyboards/blockboy/ac980mini/rules.mk:19]:
```suggestion
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/blockboy/ac980mini/ac980mini.c
Comment (drashna) [inline keyboards/blockboy/ac980mini/ac980mini.c:62]: Why bother including this then?  You'd be better off throwing this into the keymap.c file, in that case. 
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/blockboy/ac980mini/ac980mini.c
Comment (rooski15) [reply keyboards/blockboy/ac980mini/ac980mini.c]: My misunderstanding. This file was initially copied from another, recently commited keyboard folder, so I was unaware this was an issue.

**Review by drashna — APPROVED** [review]

**Comment by rooski15** [comment]
Just checking if there's any thing else I need to do here?

**Comment by drashna** [comment]
Mostly, just waiting on a second review.

File: keyboards/blockboy/ac980mini/readme.md
Comment (fauxpark) [inline keyboards/blockboy/ac980mini/readme.md:1]: Please use Unix line endings for all files.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/blockboy/ac980mini/readme.md
Comment (fauxpark) [inline keyboards/blockboy/ac980mini/readme.md:7]:
```suggestion
An F-Row-Less 1800 project with fixed layout and lots of RGB.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by rooski15** [comment]
Checking back in. Is there more I need to change here?

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**
